samples/boards/hexiwear/dock-blinky/README.rst

```diff
@@ -0,0 +1,35 @@
+.. _hexiwear-docking-station-sample:
+
+Docking Station Blinky Application
+##################################
+
+Overview
+********
+
+This example shows the LED/input lines on the Hexiwear docking station
+
+Requirements
+************
+
+A Hexiwear module and the docking station.
+
+The three right-most DIP switches (labelled LED1..3) need to be in the UP
+position (LED position).  When these are in the down position, the
+corresponding pushbuttons are usable when the GPIO is set for GPIO_DIR_IN.
```

Comment:
I'd prefer we modify one of the existing blinky samples (basic/blinky or basic/disco) rather than adding a new sample.